packages/vue/src/components/organisms/SfSidebar/SfSidebar.vue

```diff
@@ -22,25 +21,47 @@
               class="sf-heading--left sf-heading--no-underline sf-sidebar__title desktop-only"
             />
           </slot>
+          <slot
+            v-if="position === 'right'"
+            name="icon"
+            v-bind="{ close, button }"
+          >
+            <SfIcon
+              v-if="button"
+              icon="cross"
+              size="14px"
+              role="button"
+              aria-label="close"
+              class="sf-sidebar__icon desktop-only"
+              @click="close"
+            />
+          </slot>
+          <slot v-else name="circle-icon" v-bind="{ close, button }">
+            <SfCircleIcon
+              v-if="button"
+              icon-size="14px"
+              icon="cross"
+              class="sf-sidebar__circle-icon desktop-only"
+              @click="close"
+            />
+          </slot>
+        </div>
```

Comment:
we need modifier here, we cant decide about changing SfIcon to SfCircle base on position :)